### ISSUE TITLE
PIP package isntallation doesn't seem to work...

### ISSUE THREAD

**Issue by Montanaz0r** [CLARIAH/grlc#386]
Hi @c-martinez! Recently together with @rapw3k we have tried to install grlc through pip. Unfortunately, it doesn't work. Setup.py breaks with the following error:

```
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in keepalive setup command: use_2to3 is invalid.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed
```

I have tried it on both macOS and Linux (the latest Ubuntu). 

PS I was playing with this issue a bit and I was able to fix the initial error by installing a specific version of setuptools, however as a result I ended up with another dependency issue, this time greenlet extension:

```
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> greenlet
```

Could you please check and let me know if this is fixable? 

Best regards,
montanaz0r

**Comment by c-martinez** [comment]
Hey @Montanaz0r -- out of curiosity, which version of python are you using?

**Comment by Montanaz0r** [comment]
Hey, @c-martinez 3.9 and 3.10 respectively. Thanks for letting me know, I will try an older version of Python and let you know!

**Comment by Montanaz0r** [comment]
hey @c-martinez a quick update; I have tested two other solutions:

1) python 3.6 and all grlc versions below the latest one on pip (1.3.7 - this one requires python 3.7 at least). All of them failed due to the conflicts

2) python 3.7 with the latest version of grlc from pip (1.3.7). Here we got the same error that I made initial ticket with so use_2to3 is invalid...

Best regards,
montanaz0r

**Comment by c-martinez** [comment]
Hey @Montanaz0r , I've just added a fix for this issue (although not yet for Python 3.10). Once tested we can probably merge to dev branch and then to master for the next release.

If you are able to test it, that would be very helpful!

**Comment by rapw3k** [comment]
Hi @c-martinez 
i tried with python 3.9 but didnt work for me.
Problem was with use_2to3 as originally mentioned in the ticket...


```
edu240-183:grlc rap$ pip3.9 install .
DEPRECATION: Configuring installation scheme with distutils config files is deprecated and will no longer work in the near future. If you are using a Homebrew or Linuxbrew Python, please see discussion at https://github.com/Homebrew/homebrew-core/issues/76621
Processing /Users/rap/GitRepositories/grlc-clone/grlc
  Preparing metadata (setup.py) ... done
Collecting docopt==0.6.2
  Using cached docopt-0.6.2.tar.gz (25 kB)
  Preparing metadata (setup.py) ... done
Collecting docutils==0.17.1
  Using cached docutils-0.17.1-py2.py3-none-any.whl (575 kB)
Collecting Flask==1.0.2
  Using cached Flask-1.0.2-py2.py3-none-any.whl (91 kB)
Collecting Flask-Cors==3.0.6
  Using cached Flask_Cors-3.0.6-py2.py3-none-any.whl (13 kB)
Collecting gevent==1.4.0
  Using cached gevent-1.4.0.tar.gz (5.2 MB)
  Preparing metadata (setup.py) ... done
Collecting greenlet==0.4.15
  Using cached greenlet-0.4.15.tar.gz (59 kB)
  Preparing metadata (setup.py) ... done
Collecting html5lib==1.0.1
  Using cached html5lib-1.0.1-py2.py3-none-any.whl (117 kB)
Collecting isodate==0.5.4
  Using cached isodate-0.5.4.tar.gz (27 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in isodate setup command: use_2to3 is invalid.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by c-martinez** [comment]
Closed by new release [v1.3.9](https://github.com/CLARIAH/grlc/releases/tag/v1.3.9)
